zigpy/zcl/clusters/general.py

```diff
@@ -942,6 +942,27 @@ class ServerCommandDefs(BaseCommandDefs):
             direction=Direction.Client_to_Server,
         )
 
+    class ClientCommandDefs(BaseCommandDefs):
+        off: Final = ZCLCommandDef(
+            id=0x00, schema={}, direction=Direction.Server_to_Client
+        )
+        on: Final = ZCLCommandDef(
+            id=0x01, schema={}, direction=Direction.Server_to_Client
+        )
+        toggle: Final = ZCLCommandDef(
+            id=0x02, schema={}, direction=Direction.Server_to_Client
+        )
+
+    def handle_cluster_request(
+            self,
+            hdr: foundation.ZCLHeader,
+            args: list[Any],
+            *,
+            dst_addressing=None,
+    ) -> None:
+        if not hdr.frame_control.disable_default_response:
+            self.send_default_rsp(hdr.tsn, hdr.command_id, disable_default_response=False, _direction=Direction.Server_to_Client)
```

Comment:
`disable_default_response=True` should be set on a default response. I think `send_default_rsp` should do so internally.